modules/core/src/samples/kotlin/io/github/lmliam/kotventure/core/audience/TimedBossBarSamples.kt

```
@@ -0,0 +1,34 @@
+import kotlin.time.Duration.Companion.seconds
+
+internal fun timedBossBarSample() {
+    // Platform code supplies a real ticker (e.g. paperTicker(plugin)); tests use ManualTicker.
+    val ticker =
+        object : Ticker {
+            override fun repeating(
+                interval: Duration,
+                action: () -> Unit,
+            ): TickerTask =
+                object : TickerTask {
+                    override fun cancel() {}
```

detekt (1.23.8)

[warning] This empty block of code can be removed.

(detekt.empty-blocks.EmptyFunctionBlock)